docs/book/src/reference/completion.md

```diff
@@ -0,0 +1,29 @@
+# Enable shell auto completion
+
+## Prerequisites
```

Comment:
</aside>

<aside class="note">
<h1>Prerequisites for Bash</h1>

However, the completion Bash script depends on [bash-completion](https://github.com/scop/bash-completion), which means that you have to install this software first (you can test if you have bash-completion already installed.
</aside>

Reply:
Need to talk about zsh too

Reply:
@estroz Is the bash-completion a prerequisite for the zsh?